po/ru.po

```diff
@@ -8,14 +8,15 @@ msgstr ""
 "Project-Id-Version: spot\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2021-03-16 19:27+0100\n"
-"PO-Revision-Date: 2021-03-16 19:21+0100\n"
+"PO-Revision-Date: 2021-05-5 16:21+0100\n"
 "Last-Translator: Automatically generated\n"
 "Language-Team: Paul Bragin <zeusmods@protonmail.com>\n"
 "Language: ru\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=ASCII\n"
+"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
```

Comment:
Great, but you also need to update `msgstr[0]`, `msgstr[1]` and add `msgstr[2]` around line 100 to make use of the plural forms. I can't speak Russian, so I won't be able to help you with that.
All this line in the header does is that it specifies which plural form (which item from the msgstr array) should be used for different values of n (for example different numbers of selected songs).